packages/service/src/db3.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as Y from "yjs";
+import { db, pool } from "./db";
+
+test("Create more updates to documents and apply the updates in different orders.", async () => {
+  // create a yjs document
+  const ydoc1 = new Y.Doc();
+  const ydoc2 = new Y.Doc();
+
+  // make "more" updates to the document (turn into some type of array)
+  const ytext = ydoc1.getText("codemirror");
+  const ytext2 = ydoc2.getText("codemirror");
+  ytext.insert(0, "Hello, World!");
+  ytext.insert(0, "This Change!");
+  ytext.insert(0, "MojoTech");
+  ytext.insert(0, "foo 1");
+
+  const state = Y.encodeStateAsUpdate(ydoc1);
+  const state2 = Y.encodeStateAsUpdate(ydoc2);
+
+  // random order generator - produces an array of distinct random values from 0 to 3 inclusive.
+  var randArr = [];
+  while (randArr.length < 4) {
+    var num = Math.floor(Math.random() * 4);
```

Comment:
random is hard for testing. Better to test a lot of permutations of a thing than a random permutation of a thing.